sdk/tools/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ add_host_tool(mkshelllink mkshelllink/mkshelllink.c)
 add_host_tool(obj2bin obj2bin/obj2bin.c)
 target_link_libraries(obj2bin PRIVATE host_includes)
 
-add_host_tool(spec2def spec2def/spec2def.c)
```

Comment:
Why?

Reply:
moved to it's own dir?

Reply:
yes that was the question (doesn't seem to be needed as the tool is still consisting of a single source file only)